lib/streamDAO.js

```diff
@@ -247,6 +247,9 @@ module.exports = function(mongoClient){
       pre.hasProperty(datum, '_groupId');
 
       datum = _.clone(datum);
+      if (!datum.modifiedTime) {
```

Comment:
This is probably harmless, but this shouldn't be conditional, clients shouldn't be able to supply this, its ours.

I suspect the existing old logic checking for createdTime to determine if its an insert/update is also similarly flawed, but that one should crash if someone submits an insert with a createdTime, whereas this would not.